chore: adding app set id in documentation (#585)
amplitude/amplitude-docs: content/collections/api/en/attribution.md

## Considerations

- When Amplitude can't match attribution events to an existing user, they're held for up to 72 hours for potential user matching. If an event isn't logged for a matching user within 72 hours of receiving the attribution data, then Amplitude **drops the attribution data**.

> - For most of Amplitude's partners, attribution is matched to Amplitude users and events via the Advertising ID (IDFA, or ADID) or the vendor Id (IDFV or App Set Id). Therefore, you must send these identifiers for attribution requests and you must set the `idfa`, `idfv`, `adid` and `app_set_id` fields in Amplitude as the Attribution ID. 

Reviewer: ```suggestion
- For most of Amplitude's partners, Amplitude uses Advertising ID (IDFA or ADID) or vendor ID (IDFV or App Set ID) for attribution to users and events. Therefore, you must send these identifiers for attribution requests and you must set the `idfa`, `idfv`, `adid` and `app_set_id` fields in Amplitude as the Attribution ID. 

```

## Considerations

- When Amplitude can't match attribution events to an existing user, they're held for up to 72 hours for potential user matching. If an event isn't logged for a matching user within 72 hours of receiving the attribution data, then Amplitude **drops the attribution data**.

> - For most of Amplitude's partners, Amplitude uses Advertising ID (IDFA or ADID) or vendor ID (IDFV or App Set ID) for attribution to users and events. Therefore, you must send these identifiers for attribution requests and you must set the `idfa`, `idfv`, `adid` and `android_app_set_id` fields in Amplitude as the Attribution ID. 

Reviewer 2: `for attribution to users and events` should be `for attribution to match users and events`?

| `platform`       | <span class="required">Required</span>. String. Either `ios` or `android`.                                                                                         | `ios`                                                    |
| `idfa` or `idfv` | <span class="required">Required for iOS</span>. String. The Identifier for Advertiser or the Identifier for Vendor. You must include *at least* one for iOS devices. | AEBE52E7-03EE-455A-B3C4-E57283966239                     |
| `adid` or `android_app_set_id` | <span class="required">Required for Android</span>. String. The Google ADID or App Set ID, or Amazon Advertising ID for Amazon devices. You must include *at least* one for Android devices.                                          | AEBE52E7-03EE-455A-B3C4-E57283966239                     |
| `android_id`       | <span class="optional">Optional</span>. String. (Android) The Android ID                                                                                           | AEBE52E7-03EE-455A-B3C4-E57283966239                     |

> | `android_app_set_id`       | <span class="optional">Optional</span>. String. (Android) The Android App Set ID                                                                                   | AEBE52E7-03EE-455A-B3C4-E57283966239                     |

Reviewer 2: is this included in the line above?
